src/pet.js

```diff
@@ -0,0 +1,94 @@
+const START_AGE = 0;
+const MAXIMUM_FITNESS = 10;
+const MINIMUM_HUNGER = 0;
+const CRITICAL_FITNESS = 3;
+const CRITICAL_HUNGER = 5;
+const PET_DEAD_ERROR = 'Your pet is no longer alive :(';
+
+function Pet(name) {
+  this.name = name;
+  this.age = START_AGE;
+  this.hunger = MINIMUM_HUNGER;
+  this.fitness = MAXIMUM_FITNESS;
+  this.children = [];
+};
+
+Pet.prototype = {
+  get isAlive() {
+    return this.age < 30 && this.hunger < 10 && this.fitness >= 0;
```

Comment:
I would define a couple of constants for these too: MAXIMUM_AGE, MAXIMUM_HUNGER and MINIMUM_FITNESS.